src/model/GroundHeatExchangerVertical.hpp

```diff
@@ -82,8 +83,10 @@ namespace model {
     /** @name Getters */
     //@{
 
+    boost::optional<double> designFlowRate() const;
```

Comment:
This should be a double, not an optional<double>, but (sadly) keeping API stable...